### DISCUSSION TITLE
Github classroom multiple projects same name cannot delete any

### DISCUSSION THREAD

**Discussion by AJM10565** [community/community#110364] [GitHub Education]
### Hi everyone,

I'm teaching a class using github classroom for the first time, and while tinkering around with my assignments, I've entered some kind of failed state with one of my assignments. I have 2 assignments with the same name, and I'd like to delete both of them but whenever I try I get the following error:
Request not acceptable
The requested resource is unavailable in the format requested.
[Go back to classroom dashboard](https://classroom.github.com/)
For the sake of getting the class off the ground I ended up creating a 3rd version of the assignment with a longer name, and i'm using that this semester. But I'd like to be able to delete the eyesore that these two ghost projects have created in my GUI, and then rename the assignment to the name of these ghost projects. If anyone could help me resolve this issue I'd appreciate it. I anticipate someone on the backend will need to manually delete these references, but that's just my intuition.

**Comment by GreenDavidA** [comment]
I'm also experiencing this issue. My students are unable to accept the assignment when it's in this duplicated state. I created another assignment as a workaround, but I'm also unable to delete the assignments. I have two in this condition.

**Comment by samus-aran** [comment]
Hi @AJM10565 & @GreenDavidA 

Are you able to share the names of your duplicate assignments so the team can investigate?

**Reply by GreenDavidA** [reply]
CISS 222 Spring 2024 Homework 6
CISS222 Spring 2024 Homework 7

**Reply by samus-aran** [reply]
Hi @GreenDavidA We have updated those assignments so they are not duplicates. Proceed and delete either one. Let us know if you have any issues.

**Reply by GreenDavidA** [reply]
The deletion of the duplicates worked. Thank you!

**Reply by AJM10565** [reply]
@samus-aran We were on spring break all week, so I didn't see this until now, the assignment is called: Comp150Week4

[Comp150Week4](https://classroom.github.com/classrooms/155941577-ajmteaching-luc-comp150-sp2024/assignments/comp150week4)

[Comp150Week4](https://classroom.github.com/classrooms/155941577-ajmteaching-luc-comp150-sp2024/assignments/comp150week4)
<img width="1401" alt="Screenshot 2024-03-08 at 17 36 42" src="https://github.com/community/community/assets/32456394/4b00a9d1-b6b8-4c6f-ae79-d14d0f0df096">

**Reply by luis-damas-isel** [reply]
Same issue. Duplicate assignments that cannot be changed or deleted.
![image](https://github.com/community/community/assets/145167710/c379ff9c-b1a5-4b9b-98fb-444ccc3f7d7d)

When trying to delete or change, an error is displayed.
![image](https://github.com/community/community/assets/145167710/cc66840c-1bce-42f0-a95c-a83989144379)

I need both deleted.

**Reply by samus-aran** [reply]
Sorry folks. Missed these notifications! I'll pass these along to the team to manually delete if they are still there. 👍

**Reply by Lnk2past** [reply]
I am getting the same error for my classroom `33629867-rutgers-camden-scientific-computing-s24`

![image](https://github.com/community/community/assets/6456398/572fa63f-fcef-4d02-8cf4-19a01802c422)

**Comment by ruhail12** [comment]
Good post but I was wondering if you could write a litte more on this subject? I’d be very thankful if you could elaborate a little bit further. Appreciate it!	<a href="https://skupaut-katowice.pl/borowa-wies/">skup auto borowa wies</a>

**Comment by giovannivicentin** [comment]
Hello AJM10565,

It sounds like you're facing a bit of a tricky situation with GitHub Classroom, but don't worry, these kinds of issues are often solvable with a bit of guidance. The error message you're encountering suggests there might be a glitch or a specific condition in GitHub Classroom that's preventing the deletion of the duplicated assignments.

Firstly, have you tried reaching out to GitHub support directly? Given the nature of the error and your intuition about needing backend intervention, contacting GitHub support would be the most straightforward approach. They have the tools and access needed to resolve such issues safely.

In the meantime, here are a couple of steps you might consider, if you haven't already:

1. **Clear Cache and Cookies:** Sometimes, web-based services can act up due to cached data. Try clearing your browser's cache and cookies to see if that resolves the issue.

2. **Use a Different Browser or Incognito Mode:** This can help bypass session-specific issues that might be contributing to the problem.

3. **Check GitHub Status:** Ensure there are no ongoing issues with GitHub services by checking [GitHub Status](https://www.githubstatus.com/). If there's a larger issue at play, it might be affecting your ability to manage assignments.

4. **GitHub Community Forums:** While you've reached out here, which is great, consider also posting this issue in the [GitHub Community Forum](https://github.community/), specifically in the GitHub Education section. Sometimes, other educators or GitHub staff members might have encountered similar issues and can offer a workaround.

5. **Documentation and Help Resources:** Review GitHub Classroom's documentation and FAQs. There might be a section on managing assignments that could offer a clue or a direct solution to this problem.

6. **Alternative Contact Methods:** If the standard support route is slow, consider reaching out to GitHub Education on social media platforms. Companies often respond quickly to public queries.

It's great that you've found a temporary workaround by creating a third version of the assignment, but I understand the importance of tidying up your dashboard for ease of use and clarity. Hopefully, one of the above steps will help resolve the issue, but if not, GitHub support should be able to assist further.

Best of luck with your class, and I hope your experience with GitHub Classroom improves from here!

Best regards

**Comment by damiancastelao** [comment]
I resolved similar issue changing the name of the tittle of the assignment, then I could deleted
